lib/decidim/gem_manager.rb

```diff
@@ -199,8 +199,45 @@ def retry_times
         ENV.fetch("RETRY_TIMES", 10).to_i
       end
 
+      def patch_component_gemfile_generator
+        content = %(# frozen_string_literal: true
+
```

Comment:
```suggestion
<%# This file is automatically generated. Do not edit this file, it will be overwritten
If you want to modidfy it you can do it in the Decidim::GemManager.patch_component_gemfile_generator method %>
```